app/models/in_person_enrollment.rb

```diff
@@ -65,6 +65,16 @@ def self.generate_unique_id
     SecureRandom.hex(9)
   end
 
+  def due_date
+    start_date = enrollment_established_at.presence || created_at
+    start_date + IdentityConfig.store.in_person_enrollment_validity_in_days.days
+  end
```

Comment:
I think this would be misleading for enrollments that have status `establishing`. Some preference for just returning nil in that case

```suggestion
  def due_date
    nil unless established_at
    established_at + IdentityConfig.store.in_person_enrollment_validity_in_days.days
  end
```

Reply:
we were using created_at here to account for some enrollments that only have created_at defined when the enrollment status changes to pending and the user becomes ready to verify. We want to default to created_at if there's no enrollment_established_at value.

Reply:
An enrollment doesn't have a due date until it's pending though. If we use the `created_at` timestamp we're just making something up that isn't real. That date has no relevance to a user or USPS. Seems like the only reason to use `created_at` as a fallback is to return some sort of date if we ever execute that codepath, so that we're showing the user some sort of date. But if we ever execute that codepath with an enrollment that doesn't have `enrollment_established_at` then wouldn't it be a bug?

It's not a hill I want to die on but I am curious why we defined it that way in the ticket, I'm going to ask the ticket creator

Reply:
I'm fine with removing the fallback to `created_at`. Note though, that

1. the variable should be `enrollment_established_at` in sheldon-b's suggested change, and
2. the `days_to_due_date` method will have to be updated to return `nil` if `due_date` returns `nil` (otherwise, it'll return `(DateTime.now...nil).count` which is `Inifinity`).
3. the `days_remaining` method on the `ReadyToVerifyPresenter` will have to be updated to anticipate a `nil` value from `days_to_due_date` (otherwise it'll raise a `NoMethodError`).

I don't think we have to be too rigorous about protecting against this case though, because when we write the job, we shouldn't be generating emails for non-pending enrollments, which means we should never be in the circumstance where we're getting a nil value for due date.